src/pages/student/main/Dashboard/components/Recent/index.js

```diff
@@ -73,11 +74,15 @@ const Recent = ({ recentQuizzes, quizzes }) => {
               <tr>
                 <td id={style.listTable}></td>
                 <td>
-                  <Link
-                    to={`/categories/${recentQuizzes.category_id}/quizzes/${recentQuizzes.quiz_id}/questions`}
-                  >
-                    <p className={style.retake}>Retake Quiz</p>
-                  </Link>
+                  {recentQuizzes.score < passing ? (
+                    <Link
+                      to={`/categories/${recentQuizzes.category_id}/quizzes/${recentQuizzes.quiz_id}/questions`}
+                    >
+                      <p className={style.retake}>Retake Quiz</p>
+                    </Link>
+                  ) : (
+                    ''
+                  )}
```

Comment:
can you change this to a normal 'if' statement so that there is no else statement where it is just blank?